convert2rhel/actions/__init__.py

```diff
@@ -0,0 +1,619 @@
+# Copyright(C) 2016 Red Hat, Inc.
```

Comment:
Since it is a new module, should we update this license header to 2023?

Reply:
This started as a copy of checks.py and this copyright header came over with it.  In reviewing it now, I think all of the code from checks.py has been moved to individual Action plugins; all code here is new.

Note that copyright is tricky, even if you've written every line in a file, sometimes since you were inspired by the original code, the prior author(s) have some copyright claim to the new work.  In this case, since the functionality stored here is entirely different than what is in checks.py, I think it's fine.
```suggestion
# Copyright: 2023, Red Hat
```